src/lib/machineAdjustments.ts

```diff
@@ -0,0 +1,315 @@
+        translation: 'Seat'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '001799': { // Arm Curl
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8', '9'],
+        translation: 'Seat'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '001999': { // Triceps
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8', '9'],
+        translation: 'Seat'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '002199': { // Lat Pulldown
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8'],
+        translation: 'Seat'
+      },
+      {
+        name: 'leftPosition',
+        range: ['1', '2', '3', '4'],
+        translation: 'Arms'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '002399': { // Abductor
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7'],
+        translation: 'Back'
+      },
+      {
+        name: 'leftPosition',
+        range: ['1', '2', '3', '4', '5', '6'],
+        translation: 'Left Leg'
+      },
+      {
+        name: 'rightPosition',
+        range: ['1', '2', '3', '4', '5', '6'],
+        translation: 'Right Leg'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '002499': { // Adductor
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7'],
+        translation: 'Back'
+      },
+      {
+        name: 'leftPosition',
+        range: ['1', '2', '3', '4', '5', '6'],
+        translation: 'Left Leg'
+      },
+      {
+        name: 'rightPosition',
+        range: ['1', '2', '3', '4', '5', '6'],
+        translation: 'Right Leg'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '002699': { // Standing Hip
+    adjustments: [
+      {
+        name: 'seat',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8', '9', 'A'],
+        translation: 'Platform'
+      },
+      {
+        name: 'start',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8', '9'],
+        translation: 'Start'
+      },
+      {
+        name: 'leftPosition',
+        range: ['1', '2', '3', '4', '5', '6', '7'],
+        translation: 'Leg'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '002899': { /* Lower Back */
+    adjustments: [],
+    primaryAdjustmentIndex: null
+  },
+  '002999': { /* Seated Calf */
+    adjustments: [],
+    primaryAdjustmentIndex: null
+  },
+  '003298': { // Runner
+    adjustments: [
+      {
+        name: 'leftPosition',
+        range: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 'A', 'C'],
+        translation: 'Chest'
+      }
+    ],
+    primaryAdjustmentIndex: null
+  },
+  '003197': { /* Half Rack */
+    adjustments: [],
+    primaryAdjustmentIndex: null
+  }
+}
+
+export const getAvailableMachineAdjustments = (model: string) => {
```

Comment:
This needs to return a class.  You introduce a structure above, but didn't really do anything with it.